### PR TITLE
Backfill calendar.yml

### PR DESCRIPTION
**Pull request by billyroh**
:raised_hands: :date: :raised_hands:

### CODE REVIEW

File: models/calendar.yml [hunk 1]
```diff
@@ -330,14 +330,14 @@
...
+          with audio. I’ll show you how you can use this API to pull apart an
           existing song into short samples and put it back together to create a
           new song, with very little code.
     - 9:00 PM:
         emoji: 🔥
         type: Talk
-        title: 'Everything old is new again (hopefully): functional programming
-          in JavaScript'
+        title: Everything old is new again (hopefully): functional programming
+          in JavaScript
```
Comment (visnup) [inline models/calendar.yml]: Yaml needs the quotes here otherwise it gets confused with the :

**Comment by visnup** [comment]
Yay CI caught the failure!

**Comment by billyroh** [comment]
image
Fixed the string issue, but now it's saying that moment.js isn't loading properly?

**Comment by billyroh** [comment]
Oop never mind I just forgot to run `npm update`

**Pull request merged**